packages/migrate/index.js

```diff
@@ -42,12 +42,21 @@ const Migrate = {
     if (files.length === 0) return [];
 
     let migrations = files
-      .filter(file => isNaN(parseInt(path.basename(file))) === false)
       .filter(
         file =>
           path.extname(file).match(config.migrations_file_extension_regexp) !=
           null
       )
+      .filter(file => {
+        const hasNumber = Migration.getNumber(path.basename(file)) !== null;
+        if (!hasNumber) {
+          process.emitWarning(
+            `Migration file ${file} does not have a number at the beggining of the file, \
+            migration file names should match the schema \`\${\\d+}you_migration_file.js\``
+          );
+        }
```

Comment:
Neat! Can you add a test for this please?

Also, I realize this was here from before, but there might be a way to combine the two filters `migrations_file_extension_regexp`. (No need to change this, just leaving it here as a note)

Reply:
i'll do it